gradle.properties

```diff
@@ -10,4 +10,4 @@ org.gradle.caching = true
 # Versioning must follow Ragnar�k versioning convention: https://gist.github.com/JustDesoroxxx/5d4a45785ce19a6653ba99f72325c703
 id = mwc
 name = Modern Warfare Cubed
-version = 0.1-Dev-8
+version = 0.1-Dev-8.1
```

Comment:
1. Version Bumps should only happen right before posting a version to the Public
2. Have you glanced through the versioning convention, because dev builds are just a number no decimals